polynote-kernel/src/main/scala/polynote/messages/package.scala

```diff
@@ -48,13 +48,15 @@ package object messages {
   type ShortList[A] = List[A] @@ ShortTag
   object ShortList {
     def apply[A](list: List[A]): ShortList[A] =
-      if (list.size > Short.MaxValue)
+      if (list != null && list.size > Short.MaxValue)
```

Comment:
oof, where was it getting a `null` from?

Reply:
It was a mistake, just had `lazy val Nil = ShortList(Nil)` so I could say `ShortList.Nil` to get an empty ShortList. But of course that `Nil` is referencing itself... so it always ended up as `null`. This check probably isn't necessary anymore (but doesn't hurt either)

Reply:
Huh, shouldn't the compiler prevent that a `null` in that case?